frontend/src/components/Displays/ConfirmScheduleDialogs/ConfirmScheduleDialog.tsx

```diff
@@ -1,8 +1,11 @@
 import { Robot } from 'models/Robot'
```

Comment:
Is the second commit really needed since it seems to mainly just fix the previous commit? Can they not be merged?

Reply:
Wasn't sure if we wanted "confirm"-dialog or "this is insufficient values"-dialog. Wanted to keep an easy way to revert. Can squash of current design is agreed on